tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,39 @@
 """
 
 
+test_file = """
+#include "CPPCleanBearHeaderTest1.h"
+#include "CPPCleanBearHeaderTest2.h"
+
+int main() {
+    cppcleanbearfunction1();
+    cppcleanbearfunction2();
+    return 0;
+}
+"""
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearNoIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(),
+    invalid_files=(test_file,),
+    tempfile_kwargs={'suffix': '.cpp'}
+    )
+
+CPPCleanBearValidIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(test_file,),
+    invalid_files=(),
+    settings={'include_paths':  os.path.join(os.path.dirname(__file__),
+                                             'test_files',
+                                             'testCPPHeader1')+','
+              + os.path.join(os.path.dirname(__file__),
+                             'test_files',
+                             'testCPPHeader2')},
```

Comment:
Could you maybe also make the test directory names simpler? They look so complicated :3 Maybe just `headers1`, `headers2` directories and `foo.h`, `bar.h` inside them respectively? Imo they are a bit too long 😅

Reply:
Ah and the functions could also then be called `void foo()` and `void bar()` :P